opm/input/eclipse/EclipseState/Grid/RegionSetMatcher.hpp

```diff
@@ -126,17 +126,17 @@ class RegionSetMatchResult
             /// Accessible to RegionIndexRange only.
             ///
             /// \param[in] index range element value.
-            Iterator(int i) : i_{i} {}
+            explicit Iterator(int i) : i_{i} {}
 
             /// Index range element value
             int i_;
         };
 
         /// Start of Range.
-        Iterator begin() const { return { this->begin_ }; }
+        Iterator begin() const { return { Iterator(this->begin_) }; }
```

Comment:
If we're going to make the constructor explicit, then we should at least write
```C++
return Iterator { ... };
```
instead of
```C++
return { Iterator { ... } };
```
since the latter will copy-construct an `Iterator` object from the `Iterator` that gets constructed in its braced initialiser.

Reply:
yeah, just a brain prrt.